src/lib/helpers/delete.ts

```diff
@@ -24,3 +24,11 @@ export async function deleteAudio(entry: IEntry) {
     alert(`${$_('misc.error', { default: 'Error' })}: ${err}`);
   }
 }
+
+export function deleteObjectEmptyFields(ObjData: any) {
+  for (const key in ObjData) {
+    if (ObjData[key] === '' || ObjData[key] === undefined || ObjData[key] === null) {
+      delete ObjData[key];
+    }
+  }
+}
```

Comment:
This is not testable and it produces a side effect. If we are using a helper function that's simple like this I would prefer a "pure function" meaning it "does not mutate anything outside itself. A pure function does not produce side effects. Given the same inputs, a pure function will always return the same output."

Reply:
I got it!